cli/src/index.ts

```diff
@@ -302,6 +306,14 @@ export class Commander {
         '--config [PATH]',
         'load the parameters from the local configuration file, which supports json and yaml format, default path is ./mqttx-cli-config.json',
       )
+      .option(
+        '--file-write <PATH>',
+        'write the received message to the file',
```

Comment:
```suggestion
        'append received messages to a specified file',
```

easier to understand?